contracts/test/zapperGasComp.t.sol

```diff
@@ -405,6 +473,44 @@ contract ZapperGasCompTest is DevTestSetup {
         assertEq(collToken.balanceOf(B), collBalanceBeforeB, "B Coll bal mismatch");
     }
 
+    function testCannotAdjustTroveWithdrawCollAndBoldIfZapperIsNotReceiver() external {
+        uint256 collAmount1 = 10 ether;
+        uint256 collAmount2 = 1 ether;
+        uint256 boldAmount1 = 10000e18;
+        uint256 boldAmount2 = 1000e18;
+
+        IZapper.OpenTroveParams memory params = IZapper.OpenTroveParams({
+            owner: A,
+            ownerIndex: 0,
+            collAmount: collAmount1,
+            boldAmount: boldAmount1,
+            upperHint: 0,
+            lowerHint: 0,
+            annualInterestRate: MIN_ANNUAL_INTEREST_RATE,
+            batchManager: address(0),
+            maxUpfrontFee: 1000e18,
+            addManager: address(0),
+            removeManager: address(0),
+            receiver: address(0)
+        });
+        vm.startPrank(A);
+        uint256 troveId = gasCompZapper.openTroveWithRawETH{value: ETH_GAS_COMPENSATION}(params);
+        vm.stopPrank();
+
+        vm.startPrank(A);
+        // Add a remove manager for the zapper
+        gasCompZapper.setRemoveManagerWithReceiver(troveId, B, A);
```

Comment:
This call doesn't actually set the receiver in the core, right (only in the zapper)?

Receiver in core is only set in the next line AFAICS. No big deal though - was just a little confused by "Change receiver"

Reply:
Yes, that’s correct, but it’s still "change", because the receiver in core is set on opening trove.